python/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -362,24 +363,50 @@ class RandomForestClassifier(Base):
             raise ValueError("Wrong value passed in for max_features"
                              " please read the documentation")
 
-    def _get_model_info(self):
+    def _convert_to_treelite(self, task_category):
```

Comment:
why do we need the task_category variable? Isn't it known because we know this is a classifier?